content/en/docs/ops/deployment/performance-and-scalability/index.md

```diff
@@ -108,32 +108,48 @@ is busy handling the request, the worker won't start handling the next request
 
 Inside the mesh, a request traverses the client-side proxy and then the server-side
-proxy. In the default configuration of Istio {{< istio_release_name >}} (i.e. Istio with telemetry v2), the two proxies add about 3.12 ms and 3.13 ms to the 90th and 99th percentile latency, respectively, over the baseline data plane latency.
-We obtained these results using the [Istio benchmarks](https://github.com/istio/tools/tree/{{< source_branch_name >}}/perf/benchmark)
+proxy. In the default configuration of Istio {{< istio_release_name >}} (i.e. Istio with telemetry v2),
+the two proxies add about 2.76 ms and 2.88 ms to the 90th and 99th percentile latency, respectively, over the baseline data plane latency.
+After enabling `jitter`, those numbers reduced to 1.72 ms and 1.91 ms, respectively. We obtained these results using the [Istio benchmarks](https://github.com/istio/tools/tree/{{< source_branch_name >}}/perf/benchmark)
 for the `http/1.1` protocol, with a 1 kB payload at 1000 requests per second using 16 client connections, 2 proxy workers and mutual TLS enabled.
 
-In upcoming Istio releases we are moving `istio-policy` and `istio-telemetry` functionality into the proxy as `TelemetryV2`.
-This will decrease the amount data flowing through the system, which will in turn reduce the CPU usage and latency.
+{{< image width="90%"
+    link="latency_p90_fortio_without_jitter.svg"
+    alt="P90 latency vs client connections"
+    caption="P90 latency vs client connections without jitter"
+>}}
+
+{{< image width="90%"
+    link="latency_p99_fortio_without_jitter.svg"
+    alt="P99 latency vs client connections"
+    caption="P99 latency vs client connections without jitter"
+>}}
 
 {{< image width="90%"
-    link="latency_p90_fortio.svg"
+    link="latency_p90_fortio_with_jitter.svg"
     alt="P90 latency vs client connections"
-    caption="P90 latency vs client connections"
+    caption="P90 latency vs client connections with jitter"
 >}}
 
 {{< image width="90%"
-    link="latency_p99_fortio.svg"
+    link="latency_p99_fortio_with_jitter.svg"
     alt="P99 latency vs client connections"
-    caption="P99 latency vs client connections"
+    caption="P99 latency vs client connections with jitter"
 >}}
 
 - `baseline` Client pod directly calls the server pod, no sidecars are present.
-- `none-both` Istio proxy with no Istio specific filters configured.
-- `telemetryv2-both` client and server sidecars are present with telemetry v2 `nullvm` configured by default.
-- `mixer-both` Client and server sidecars are present with mixer configured.
+- `none_both` Istio proxy with no Istio specific filters configured.
+- `v2-stats-wasm_both` Client and server sidecars are present with telemetry v2 `v8` configured.
+- `v2-stats-nullvm_both` Client and server sidecars are present with telemetry v2 `nullvm` configured by default.
+- `v2-sd-full-nullvm_both` Export stackdriver metrics, access logs and edges with telemetry v2 `nullvm` configured.
```

Comment:
No need to include sd in the result.

Reply:
If you don't remove this line, it has a lint spelling error:
```suggestion
- `v2-sd-full-nullvm_both` Export Stackdriver metrics, access logs and edges with telemetry v2 `nullvm` configured.
```

Reply:
Offline discussed, will keep the stackdriver numbers.